sdk/lib/drivers/ip/network/ports.c

```diff
@@ -36,8 +36,11 @@ VOID DeallocatePort( PPORT_SET PortSet, ULONG Port ) {
     KIRQL OldIrql;
 
     Port = htons(Port);
-    ASSERT(Port >= PortSet->StartingPort);
-    ASSERT(Port < PortSet->StartingPort + PortSet->PortsToOversee);
```

Comment:
DeallocatePort can only be called after a port has been created?
In that case it's perfectly fine to assert on an invalid port, because that should never happen

Reply:
Yes that's true now that the proposed change fully allow full range to be supported up to 0xFFFF. @muthu90tech : please re-enable these 2 ASSERTs